zipkin-lens/src/components/TracePage/TraceSummaryHeader.jsx

```diff
@@ -104,6 +104,51 @@ const TraceSummaryHeader = React.memo(({ traceSummary, rootSpanIndex }) => {
     ? config.logsUrl.replace('{traceId}', traceSummary.traceId)
     : undefined;
 
+  const archiveWriteUrl = (config.archiveWriteUrl && traceSummary)
+    ? config.archiveWriteUrl
+    : undefined;
+
+  const archiveReadUrl = (config.archiveReadUrl && traceSummary)
+    ? config.archiveReadUrl.replace('{traceId}', traceSummary.traceId)
+    : undefined;
+
+  const archiveClick = useCallback(() => {
+    // We don't store the raw json in the browser yet, so we need to make an
+    // HTTP call to retrieve it again.
+    fetch(`${api.TRACE}/${traceSummary.traceId}`)
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error('Failed to fetch trace from backend');
+        }
+        return response.json();
+      })
+      .then((json) => {
+        fetch(archiveWriteUrl, {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify(json),
+        })
+        .then((response) => {
+          if (!response.ok || response.status != 202) {
```

Comment:
I think some clones incorrectly use 200

Reply:
I added this check on the status code after I mistakenly set the url to `localhost:9412/zipkin`. The homepage happily returned 200 to my POST request and I spent a while trying to figure out why the trace wasn't saved.

I can remove it though

Reply:
we can wait until someone asks for 200 support. it is not following our spec to use 200 anyway.

I'm really surprised homepage accepted POST! open an issue?